File: 01-FUNDAMENTOS_DE_PROGRAMACAO/Dart/exercicio1.dart
// 1 - Faça um algoritmo que receba o peso de uma pessoa, calcule e mostre:
// a. O novo peso quando a pessoa engorda 15% do peso digitado
// b. O novo peso quando a pessoa engorda 20% do peso digitado

import 'dart:io';

main(List<String> args) {
  print('Digite seu peso: ');
  var peso = double.parse(stdin.readLineSync()!);
  print('O peso da pessoa quando engorda 15% é: ${peso + peso * 0.15}');
  print('O peso da pessoa quando engorda 20% é: ${peso + peso * 0.20}');
}

File: 01-FUNDAMENTOS_DE_PROGRAMACAO/JavaScript/Exercicio1.js
/*
1 - Faça um algoritmo que receba o peso de uma pessoa, calcule e mostre:
a. O novo peso quando a pessoa engorda 15% do peso digitado
b. O novo peso quando a pessoa engorda 20% do peso digitado
*/

//Em javaScript para comandos de entrada precisa do HTML por isso será feito sem entrada do usuário
//exemplo de dados de entrada ===>   var num = prompt('DIgite o valor de num: ') 

// const peso = prompt('Digite seu peso: ')
//const peso = window.prompt('Digite seu peso: ')

const peso=18
novopeso15 = peso+(peso*0.15)
novopeso20 = peso+(peso*0.20)

console.log('Novo peso com 15%: ',novopeso15)
console.log('Novo peso com 20%: ',novopeso20)